src/openrct2/ride/VehiclePaint.cpp

```diff
@@ -3045,12 +2946,12 @@ static constexpr const vehicle_sprite_func PaintFunctionsByPitch[] = {
     VehiclePitchFlat, // Half Helix Down Small
     VehiclePitchFlat, // Quarter Helix Up
     VehiclePitchFlat, // Quarter Helix Down
-    VehiclePitchDiagUp12,
-    VehiclePitchDiagUp25,
-    VehiclePitchDiagUp60,
-    VehiclePitchDiagDown12,
-    VehiclePitchDiagDown25,
-    VehiclePitchDiagDown60,
+    VehiclePitchUp8,
```

Comment:
i feel it should have kept diag in the name what was your reasoning for removing that bit?

Reply:
With this refactor a new track element could use these slopes for non-diagonal tracks (the sprites would start on the cardinal direction), so I named them to their geometric angle instead of diag.